src/Nethermind/Nethermind.Facade/Filters/FilterLog.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Nethermind.Facade.Filters
 {
-    public class FilterLog
+    public class FilterLog : IFilterLog
```

Comment:
Why do we need `IFilterLog` as an interface? I don't see any other class implementing it.

Reply:
I added this because I had a test that was faking out the functionality of FilterLog, but it's no longer being used. I could remove it but I don't see any reason not to have an interface?

Reply:
Remove please!